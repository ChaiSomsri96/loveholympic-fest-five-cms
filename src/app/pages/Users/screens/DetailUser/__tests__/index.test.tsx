import * as React from 'react';
import { render } from '@testing-library/react';

import { DetailUser } from '..';

jest.mock('react-i18next', () => ({
  useTranslation: () => {
    return {
      t: str => str,
      i18n: {
        changeLanguage: () => new Promise(() => {}),
      },
    };
  },
}));

describe('<DetailUser  />', () => {
  it('should match snapshot', () => {
    const loadingIndicator = render(<DetailUser />);
    expect(loadingIndicator.container.firstChild).toMatchSnapshot();
  });
});
