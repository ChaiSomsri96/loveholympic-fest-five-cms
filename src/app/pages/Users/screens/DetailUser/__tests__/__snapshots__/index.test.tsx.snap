// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<DetailUser  /> should match snapshot 1`] = `
.c0 {
  height: 100%;
  padding: 1rem;
  background-color: #ecf0f5;
}

.c0 .wrap-chart {
  background-color: #fff;
  padding: 1rem;
  margin: 0;
}

.c0 .wrap-chart .average {
  display: block;
  margin-top: 2rem;
  text-align: center;
  color: #ff4646;
  font-size: 24px;
}

.c0 .wrap-chart .result-test p {
  color: #3c8dbc;
  font-size: 18px;
}

.c0 .info-user {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
}

<div
  class="c0"
>
  <div
    class="wrap-chart row"
  >
    <div
      class="col-sm-6 col-md-6"
    >
      <canvas
        height="150"
        width="300"
      />
      <b
        class="average"
      >
        Average 6.5
      </b>
    </div>
    <div
      class="col-sm-5"
    >
      <div
        class="result-test pt-4"
      >
        <b>
          Result Test
        </b>
        <p>
          1.pographers and the like. Some people hate it
        </p>
        <p>
          2.pographers and the like. Some people hate it
        </p>
        <p>
          3.pographers and the like. Some people hate it
        </p>
        <p>
          4.pographers and the like. Some people hate it
        </p>
        <p>
          5.pographers and the like. Some people hate it
        </p>
      </div>
    </div>
  </div>
</div>
`;
